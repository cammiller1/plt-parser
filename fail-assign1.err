Stored value type does not match pointer operand type!
  store double 3.200000e+00, i32* @i
 i32LLVM ERROR: Broken module found, compilation aborted!
