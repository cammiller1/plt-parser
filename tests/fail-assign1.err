Fatal error: exception Failure("illegal assignment boolean = int in i = 1")