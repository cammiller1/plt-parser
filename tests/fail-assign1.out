Fatal error: exception Failure("illegal assignment int = float in i = 3.2")